src/react/ui-elements/Veeam/ VeeamCapacityModal.tsx

```diff
@@ -0,0 +1,143 @@
+import {
```

Comment:
nit: the file name starts with a space

Reply:
Fixed in 49e7d8dae934e9182e815b5ad0f47c51712c374b